code/modules/recycling/sortingmachinery.dm

```diff
@@ -185,6 +188,15 @@
 			wrapped_item.AddComponent(/datum/component/pricetag, sticker.payments_acc, sticker.cut_multiplier)
 		update_appearance()
 
+	if(istype(item, /obj/item/boxcutter))
+		if( (item.icon_state == "boxcutter_on"))
```

Comment:
The boxcutter should have an 'on' var instead, looking at the icon state is a bad idea,